astropy/table/tests/test_mixin.py

```diff
@@ -176,22 +176,85 @@ def test_io_time_write_fits(tmpdir, table_types):
         assert (tm[name] == t[name]).all()
 
     # Test for conversion of time data to its value, as defined by its format
-    for scale in time.TIME_SCALES:
+    for scale in time.STANDARD_TIME_SCALES:
         for ab in ('a', 'b'):
             name = ab + scale
             t[name].info.serialize_method['fits'] = 'formatted_value'
 
     t.write(filename, format='fits', overwrite=True)
     tm = table_types.read(filename, format='fits')
 
-    for scale in time.TIME_SCALES:
+    for scale in time.STANDARD_TIME_SCALES:
         for ab in ('a', 'b'):
             name = ab + scale
 
             assert not isinstance(tm[name], time.Time)
             assert (tm[name] == t[name].value).all()
 
 
+@pytest.mark.parametrize('table_types', (Table, QTable))
+def test_io_time_write_fits_local(tmpdir, table_types):
+    """
+    Test that table with Time mixin columns can be written by io.fits.
```

Comment:
I think the docstring was just copy-pasted from the function above. Could you update it? (i.e., state that this tests the `local` time scale, and describe why it cannot be included above)?